### ISSUE TITLE
Doesn't stop when user moves the mouse or types.

### ISSUE THREAD

**Issue by HubGid** [sw3103/movemouse#25]
You can easily reproduce this by setting the distance to 9999. (random)
When 'Move Mouse' is moving the pointer and when I move the mouse during that 'Move Mouse' doesn't stop and keeps moving until the distance is completed.

Please let 'Move Mouse' stop all it's activity immediately the moment the user moves the mouse.

PS
Please also allow to randomize the movement speed during the movement itself.
Even some stops once in a while just like a real user does.

**Comment by sw3103** [comment]
You are quite right. Move Mouse will only allow pauses _between_ Actions, not during.

My recommendation would be to break down any long running Actions such as this into smaller ones if this is functionality you require.
